.github/workflows/main.yml

```diff
@@ -15,19 +15,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest]  # TODO add windows and fix bash scripts (conda not found)
+        os: [ubuntu-latest, windows-latest]
         python_version: ["3.7", "3.8"]
     steps:
     - name: checkout
       uses: actions/checkout@v1
-
+    - uses: conda-incubator/setup-miniconda@v1
+      with:
+        auto-update-conda: true
+        python-version: ${{ matrix.python_version }}
+        activate-environment: "ibis-dev"
+        environment-file: "environment.yml"
     - name: set up environment
-      run: ./ci/setup_env.sh "${{ matrix.python_version }}" "$BACKENDS"
-      shell: bash
+      run: |
+        conda activate ibis-dev
+        ./ci/setup_env.sh "${{ matrix.python_version }}" "$BACKENDS"
+      shell: bash -l {0}
```

Comment:
Why is this required?

Reply:
via https://github.com/conda-incubator/setup-miniconda#important

> Bash shells do not use ~/.profile or ~/.bashrc so these shells need to be explicitely declared as shell: bash -l {0} on steps that need to be properly activated. This is because bash shells are executed with bash --noprofile --norc -eo pipefail {0} thus ignoring updated on bash profile files made by conda init bash. See Github Actions Documentation and thread.